hw05-autoLogging/src/main/java/ru/ProxyInvocationHandler.java

```diff
@@ -0,0 +1,39 @@
+package ru;
+
+import java.lang.reflect.InvocationHandler;
+import java.lang.reflect.Method;
+import java.util.Arrays;
+import java.util.HashSet;
+
+class ProxyInvocationHandler implements InvocationHandler {
+    private final TestLoggingInterface clazz;
+    private final HashSet<Method> listLogMethod = new HashSet<>();
```

Comment:
Лучше тип объявлять как интерфейс:
```suggestion
    private final Set<Method> listLogMethod = new HashSet<>();
```